Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM node:8
+FROM node:12
 
 WORKDIR /tmp
-RUN wget http://downloadarchive.documentfoundation.org/libreoffice/old/6.3.3.1/deb/x86_64/LibreOffice_6.3.3.1_Linux_x86-64_deb.tar.gz -O libo.tar.gz
+RUN wget http://downloadarchive.documentfoundation.org/libreoffice/old/6.4.5.2/deb/x86_64/LibreOffice_6.4.5.2_Linux_x86-64_deb.tar.gz -O libo.tar.gz
 RUN apt update \
   && apt install -y libxinerama1 libfontconfig1 libdbus-glib-1-2 libcairo2 libcups2 libglu1-mesa libsm6 unzip \
   && tar -zxvf libo.tar.gz
-WORKDIR LibreOffice_6.3.3.1_Linux_x86-64_deb/DEBS
+WORKDIR LibreOffice_6.4.5.2_Linux_x86-64_deb/DEBS
```

Comment:
Workdirs should not be used like this, I prefer this example:

```Dockerfile
FROM node:lts

WORKDIR /carbone-api

RUN apt update \
  && apt install -y libxinerama1 libfontconfig1 libdbus-glib-1-2 libcairo2 libcups2 libglu1-mesa libsm6 unzip

RUN wget http://downloadarchive.documentfoundation.org/libreoffice/old/6.4.5.2/deb/x86_64/LibreOffice_6.4.5.2_Linux_x86-64_deb.tar.gz -O libo.tar.gz \
    && tar -zxvf libo.tar.gz

RUN cd LibreOffice_6.4.5.2_Linux_x86-64_deb/DEBS && dpkg -i *.deb

COPY . /carbone-api

RUN yarn install --production

CMD node index

```

Reply:
I could agree just : what's the point but adding a slow `COPY` step ?

Reply:
index.js has to be copied into container right?

Reply:
Sorry misread your proposal. Now I can say I disagree 😛 

From [dockerfile_best-practices](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/) :
```
For clarity and reliability, you should always use absolute paths for your WORKDIR.
Also, you should use WORKDIR instead of proliferating instructions like RUN cd … && do-something, which are hard to read, troubleshoot, and maintain.
```

Reply:
Ok :DDD Agree anyway we can use that node:lts and rest :smile: 
Thank you!